public/app/features/query/state/runRequest.ts

```diff
@@ -176,10 +175,11 @@ export function callQueryMethod(
   request: DataQueryRequest,
   queryFunction?: typeof datasource.query
 ) {
-  // If the datasource has defined a default query, make sure it's applied if the query is empty
-  request.targets = request.targets.map((t) =>
-    queryIsEmpty(t) ? { ...datasource?.getDefaultQuery?.(CoreApp.PanelEditor), ...t } : t
```

Comment:
what is the reason for removing `queryIsEmpty`?

Reply:
It wasn't really doing anything, since 
1. if fields returned from getDefaultQuery will be overwritten by "t" anyway
2. if the getDefaultQuery method is undefined, it will just return the "t" anyway.